sdk/python/feast/infra/online_stores/dynamodb.py

```diff
@@ -199,26 +200,59 @@ def online_read(
             config: The RepoConfig for the current FeatureStore.
             table: Feast FeatureView.
             entity_keys: a list of entity keys that should be read from the FeatureStore.
+            batch_size: the number of items to send in a batch_get_item request to DynamoDB.
+                DynamoDB record size limit is 400kb and can retrieve 16MB per call, it is recommended
+                to set batch_size value less than 40 to avoid ``UnprocessedKeys`` and
+                ``ValidationException`` errors.
         """
+        if batch_size > 40:
```

Comment:
if we don't let user specify batch_size, then this is not needed.

Reply:
got it